### PR TITLE
iptables-persistent: add recipe

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: recipes-debian/iptables/iptables-persistent_debian.bb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+require recipes-debian/sources/iptables-persistent.inc
+DEBIAN_PATCH_TYPE="quilt"
+DEBIAN_QUILT_PATCHES = ""
+
+RDEPENDS_${PN} = "iptables"
+
+do_install() {
+    oe_runmake 'DESTDIR=${D}' install
+
+    install ${S}/plugins/* ${D}/usr/share/netfilter-persistent/plugins.d/.
```
Comment (masami256) [inline recipes-debian/iptables/iptables-persistent_debian.bb]: You can use ${datadir} as /usr/share. such variables are defined in [bitbake.conf](http://git.yoctoproject.org/cgit.cgi/poky/plain/meta/conf/bitbake.conf?h=warrior)

Comment (yuwata) [reply recipes-debian/iptables/iptables-persistent_debian.bb]: Thanks. Updated.

Comment (masami256) [reply recipes-debian/iptables/iptables-persistent_debian.bb]: LGTM :)

**Comment by masami256** [comment]
thank you!

**Pull request merged**